gotocompany/compass/v1beta1/service.proto

```diff
@@ -810,6 +810,10 @@ message SearchAssetsRequest {
       description: "offset parameter defines the offset from the first result you want to fetch"
     }
   ];
+
+  bool is_column_search = 8 [ (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "search column names instead of table urn or name."
+  }];
```

Comment:
given search does not need to know the type of asset to query for, wouldn't this field specifically tie up with table type asset ?

Reply:
Agreed. Added a generic map[string]bool to handle this.